app/views/hummingbird_survey/common/survey/_show_question.html.haml

```diff
@@ -2,4 +2,7 @@
   .question-text
     = item_options["label"]
   .response
-    = item_options["value"]
+    - if item_options["value"].include?("data:image")
```

Comment:
@Onikoroshi because I couldn't sort the question type, I did this. Feels like a hack to a degree, but wanted to get it in so we can improve on it.